lib/detect-libc.js

```diff
@@ -95,10 +83,10 @@ const familyFromCommand = (out) => {
 };
 
 const getFamilyFromLddContent = (content) => {
-  if (content.includes(MUSL_ON_LDD)) {
+  if (content.includes(MUSL)) {
```

Comment:
```suggestion
  if (content.includes('musl')) {
```

@lovell Maybe inline this one to make it consistent? I'm worried someone is going to look back years from now and think that the missing quotes is a typo.

Reply:
"someone... years from now" = me next week

Reply:
@lovell There's a [quote about this](https://twitter.com/styfle/status/1187715719846338560) 😆 

While I have your attention, do you think you could respond to https://github.com/lovell/sharp/pull/3522 when you have a moment? 🙏